ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# HSC Re-Run: Making Forced Photometry Light Curves from Scratch\n",
+    "\n",
+    "<br>Owner: **Justin Myles** ([@jtmyles](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@jtmyles))\n",
+    "<br>Last Verified to Run: **N/A -- in development**\n",
```

Comment:
Even in development we need the notebooks to run at all times - in fact _you_ need the notebook to run at all time, so you can test that the code you are writing works!